lighthouserc.cjs

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+module.exports = {
+    ci: {
+        collect: {
+            url: [
+                'http://localhost:8000/',
```

Comment:
Do the docs always use 8000 or does it fallback to an open port if that is in use? I ask because you might need to use an environment variable or something else here.

Reply:
Docs will always use 8000 as it is 11ty default port